### PR TITLE
[MAISTRA-592] Kiali/Jaeger operator docs

### PR DESCRIPTION
**Pull request by brian-avery**

### CODE REVIEW

File: topics/docs/snippets/install_jaeger.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+[source, bash]
+----
+$ oc new-project observability # create the project for the jaeger operator
+$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.12.1/deploy/crds/jaegertracing_v1_jaeger_crd.yaml
+$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.12.1/deploy/service_account.yaml
+$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.12.1/deploy/role.yaml
+$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.12.1/deploy/role_binding.yaml
+$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.12.1/deploy/operator.yaml
+----
```
Comment (jwendell) [inline topics/docs/snippets/install_jaeger.adoc]: These should be 1.13.1

File: topics/docs/snippets/remove_jaeger.adoc [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+[source, bash]
+----
+$ oc delete -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.12.1/deploy/crds/jaegertracing_v1_jaeger_crd.yaml
+$ oc delete -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.12.1/deploy/service_account.yaml
+$ oc delete -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.12.1/deploy/role.yaml
+$ oc delete -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.12.1/deploy/role_binding.yaml
+$ oc delete -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.12.1/deploy/operator.yaml
+$ oc delete project observability
+----
```
Comment (jwendell) [inline topics/docs/snippets/remove_jaeger.adoc]: These should be 1.13.1

**Review by jwendell — CHANGES_REQUESTED** [review]

File: topics/docs/getting_started/install.adoc [hunk 3]
```diff
@@ -83,6 +83,14 @@ Make the following changes on each node within your installation
 sysctl vm.max_map_count=262144
 ```
 == Installing Maistra
+=== Installing the Kiali Operator
```
Comment (JStickler) [inline topics/docs/getting_started/install.adoc]: Do we need to explicitly point out that the operators must be installed in a particular order?

File: topics/docs/snippets/install_jaeger.adoc [hunk 4]
```diff
@@ -0,0 +1,24 @@
+
+As of the Maistra 0.12 release, Jaeger now relies on the presence of the Jaeger
```
Comment (JStickler) [inline topics/docs/snippets/install_jaeger.adoc]: And here it is.  This should probably be moved up under the "Installing Maistra" heading (or possibly duplicated in both places?).

File: topics/docs/snippets/install_kiali.adoc [hunk 5]
```diff
@@ -0,0 +1,15 @@
+As of the Maistra 0.12 release, Kiali now relies on the presence of the Kiali operator being installed before Maistra. To install the operator, execute the following as a privileged user:
```
Comment (JStickler) [inline topics/docs/snippets/install_kiali.adoc]: Does Jaeger have to be installed before Kiali?

**Review by JStickler — COMMENTED** [review]
Couple of questions about operator installation order.

File: topics/docs/snippets/install_kiali.adoc (hunk 5 above)
Comment (brian-avery) [reply topics/docs/snippets/install_kiali.adoc]: It shouldn't have to be.

Comment (JStickler) [reply topics/docs/snippets/install_kiali.adoc]: I just wanted to make sure that if there was some dependency that required any of these pieces to be installed in a particular order, that we capture it in the docs.  I already had "Kiali before Istio" in my notes, but wanted to check if Jaeger had to be installed before Kiali or not.  Thanks!

**Review by jwendell — APPROVED** [review]

File: topics/docs/getting_started/install.adoc (hunk 3 above)
Comment (brian-avery) [reply topics/docs/getting_started/install.adoc]: As long as Kiali and Jaeger are installed before the istio-operator, it should be fine.

File: topics/docs/snippets/install_kiali.adoc (hunk 5 above)
Comment (brian-avery) [reply topics/docs/snippets/install_kiali.adoc]: No problem.

File: topics/docs/snippets/install_jaeger.adoc (hunk 4 above)
Comment (brian-avery) [reply topics/docs/snippets/install_jaeger.adoc]: There's a notice two lines down from the header. Could remove this and make it a warning for both operators directly below the  Installing Maistra header.

Comment (brian-avery) [reply topics/docs/snippets/install_jaeger.adoc]: https://deploy-preview-58--maistra.netlify.com/docs/getting_started/install/#_installing_the_kiali_operator

**Pull request merged**